xtask/src/ast_src.rs

```diff
@@ -607,55 +1972,165 @@ pub(crate) const AST_SRC: AstSrc = AstSrc {
             const_args: [ConstArg],
             T![>]
         }
+
+        /// Type argument that is passed at generic instantiation site.
+        ///
+        /// ```
+        /// use foo::<'a, ❰ u64 ❱, ❰ bool ❱, Item = Bar, 42>::baz;
+        /// ```
+        ///
+        /// [Reference](https://doc.rust-lang.org/reference/paths.html#paths-in-expressions)
         struct TypeArg { TypeRef }
+
+        // TODO: verify inline type bounds example
+        /// Associated type argument that is passed at generic instantiation site.
+        /// ```
+        /// use foo::<'a, u64, bool, ❰ Item = Bar ❱, 42>::baz;
```

Comment:
you can't have type args in a `use`

Reply:
Thanks, amended this in 2af6b4b67ea72b7c8365eb1bc719c896b030123d